text/0032-nft-standard.md

```diff
@@ -69,14 +69,43 @@ The metadata should have the following JSON structure:
         "type": "string",
         "description": "A URL pointing to an immutable resource with a digital asset to which this token represents"
     },
+    "collection": {
```

Comment:
Should we mark the fields that are required or optional? This would be good now that the standard has gotten more complex.